pkg/server/container_execsync.go

```diff
@@ -120,6 +127,8 @@ func (c *criContainerdService) execInContainer(ctx context.Context, id string, o
 		return nil, fmt.Errorf("failed to create exec %q: %v", execID, err)
 	}
 	defer func() {
+		// TODO(random-liu): There is a containerd bug here containerd#1376, revisit this
+		// after that is fixed.
 		if _, err := process.Delete(ctx); err != nil {
```

Comment:
As is mentioned in the containerd issue https://github.com/containerd/containerd/issues/1376, it doesn't work.